packages/nestjs-retry/package.json

```diff
@@ -60,8 +60,8 @@
     "@geersch/retry": "workspace:^"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0 || ^10.0.4",
-    "@nestjs/core": "^9.0.0 || ^10.0.4",
-    "reflect-metadata": "^0.1.12"
+    "@nestjs/common": "^10.0.0",
```

Comment:
There's a bug in NestJS 10 which affects 10.0.3 and lower. That's why the minimum version was 10.0.4.

It works fine with NestJS >= 9.x.x and >= 10.0.4 so you can leave those peer dep requirements.

https://github.com/nestjs/nest/issues/11923

Reply:
right, makes sense 👍